internal/testing/testdata/testdata.go

```diff
@@ -1033,27 +1033,17 @@ var (
 			DepPkg:          cdxResteasyPack,
 			DepPkgMatchFlag: model.MatchFlags{Pkg: model.PkgMatchTypeSpecificVersion},
 			IsDependency: &model.IsDependencyInputSpec{
-				DependencyType: model.DependencyTypeUnknown,
-				VersionRange:   "2.13.4.Final",
-				Justification:  isDepJustifyTopPkgJustification,
-			},
-		},
-		{
-			Pkg:             cdxTopQuarkusPack,
```

Comment:
This should be put back in but instead, this would be marked as `DependencyTypeIndirect` as `cdxReactiveCommonPack` is a transitive dependency of  `cdxTopQuarkusPack` as we observe from:

```
 "dependencies" : [
    {
      "ref" : "pkg:maven/org.acme/getting-started@1.0.0-SNAPSHOT?type=jar",
      "dependsOn" : [
        "pkg:maven/io.quarkus/quarkus-resteasy-reactive@2.13.4.Final?type=jar"
      ]
    },
    {
      "ref" : "pkg:maven/io.quarkus/quarkus-resteasy-reactive@2.13.4.Final?type=jar",
      "dependsOn" : [
        "pkg:maven/io.quarkus/quarkus-resteasy-reactive-common@2.13.4.Final?type=jar"
      ]
    }
```